main.js

```diff
@@ -11,24 +11,52 @@ const removeClass = (el, oldClass) => {
   ).join(" ");
 };
 
+const createElemWithProps = (elem, properties) => {
+  let element = document.createElement(elem);
+
+  for (let prop in properties) {
+    element[prop] = properties[prop];
+  }
+
+  return element;
+};
+
+const removeElem = (elem, parent) => {
+  parent.removeChild(elem);
+};
+
+const removeAllElements = elements => {
+  const allElements = elements;
+  for (i=0; i < allElements.length; i++) {
```

Comment:
Yea it was really weird. Kept giving me the index of each element but as a
string, rather than the dom element.  

![](https://link.nylas.com/open/6hy0p9pbfmrqd5xtd6chwq447/local-f8eb030a-
db78?r=cmVwbHkrMDA1NjQyODlhN2Q3NGYwYzFhOTFhMDEyZDMwOWNjNDA1YjMyMDkwMjdhNzAzOWIzOTJjZjAwMDAwMDAxMTNiMGFhYmI5MmExNzBjZTA0NTFmMTY4QHJlcGx5LmdpdGh1Yi5jb20=)